src/taipy/config/exceptions/exceptions.py

```diff
@@ -28,3 +28,7 @@ class InvalidConfigurationId(Exception):
 
 class ConfigurationUpdateBlocked(Exception):
     """The configuration is being blocked from update by other Taipy services."""
+
+
+class ConflictedConfigurationError(Exception):
+    """There is conflicted between 2 _Config found by the _ConfigComparator."""
```

Comment:
The message must help the user fix the issue. He/she does not know what is an _Congi nor a _ConfigComparator.

```suggestion
    """ Conflicts have been detected between the current and previous Configurations."""
```